gossip/src/cluster_info.rs

```diff
@@ -144,6 +145,11 @@ const MIN_STAKE_FOR_GOSSIP: u64 = solana_sdk::native_token::LAMPORTS_PER_SOL;
 /// Minimum number of staked nodes for enforcing stakes in gossip.
 const MIN_NUM_STAKED_NODES: usize = 500;
 
+// Must have at least one socket to monitor the TVU port
+// The unsafes are safe because we're using fixed, known non-zero values
+pub const MINIMUM_NUM_TVU_SOCKETS: NonZeroUsize = unsafe { NonZeroUsize::new_unchecked(1) };
+pub const DEFAULT_NUM_TVU_SOCKETS: NonZeroUsize = unsafe { NonZeroUsize::new_unchecked(8) };
```

Comment:
why are we using unsafe here? Can we just do something like:
```
pub const MINIMUM_NUM_TVU_SOCKETS: NonZeroUsize = NonZeroUsize::new_unchecked(1).unwrap();
pub const DEFAULT_NUM_TVU_SOCKETS: NonZeroUsize = NonZeroUsize::new_unchecked(8).unwrap();
```
we know the `unwrap()` is safe here

Reply:
oh i misread what you said. `the unsafes are safe...` . is this just a preference then?